.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - '*'
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
```

Comment:
Can we unify build.yml into this?

Reply:
Unified, the `golangci-lint` should be in a separate step though since it utilizes caching to provide better performance.